test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/AuthenticationTests.cs

```diff
@@ -26,8 +26,7 @@ public class AuthenticationTests
         [InlineData(AuthenticationSchemes.Negotiate | AuthenticationSchemes.NTLM | /*AuthenticationSchemes.Digest |*/ AuthenticationSchemes.Basic)]
         public async Task AuthTypes_AllowAnonymous_NoChallenge(AuthenticationSchemes authType)
         {
-            string address;
-            using (Utilities.CreateHttpAuthServer(authType, AllowAnoymous, out address, httpContext =>
+            using (var server = Utilities.CreateDynamicHost(string.Empty, authType, AllowAnoymous, out var address, out var baseAddress, httpContext =>
```

Comment:
Nope, removed, also hardcoded the string.Empty basePath in this new overload since it wasn't being set anywhere either.